test/Microsoft.Health.Fhir.Stu3.Tests.E2E/Microsoft.AspNetCore.Mvc.Testing.targets

```diff
@@ -0,0 +1,37 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The content of this file is the same as the one in R4 folder. Can this be put on root level (or create a targets folder or something) and shared from there?

Reply:
The content of this file is the same as the one in R4 folder. Can this be put on root level (or create a targets folder or something) and shared from there?